packages/components/Notification/src/Notification.tsx

```diff
@@ -66,17 +66,18 @@ export const Notification = compose<NotificationType>({
     const sizeClass = useSizeClassIOS_DO_NOT_USE();
     const onActionPress = userProps.onActionPress;
 
-    const rootStyle: ViewStyle = useMemo(() => {
+    const rootStyle: ViewProps = useMemo(() => {
       const marginHorizontal = isBar ? 0 : 16;
       if (sizeClass === 'regular' && !isBar) {
-        return { alignSelf: 'center', marginHorizontal: marginHorizontal, width: width };
+        return { style: { alignSelf: 'center', marginHorizontal: marginHorizontal, width: width } };
       } else {
-        return { marginHorizontal: marginHorizontal };
+        return { style: { marginHorizontal: marginHorizontal } };
       }
     }, [isBar, width]);
-    const messageStyle: ViewStyle = useMemo(() => {
+
+    const messageStyle: ViewProps = useMemo(() => {
```

Comment:
messageStyle should stay as {alignSelf} since the result is being passed into the style prop of the message slot.

Reply:
Yep, sorry